Src/Fido2.Models/Objects/AttestationVerificationSuccess.cs

```diff
@@ -13,5 +14,7 @@ public class AttestationVerificationSuccess : AssertionVerificationResult
         public Fido2User User { get; set; }
         public string CredType { get; set; }
         public System.Guid Aaguid { get; set; }
+        public X509Certificate2 AttestationCertificate { get; set; }
```

Comment:
Add nullability annotations.

```csharp
public X509Certificate2? AttestationCertificate
```

Reply:
Will do, didn't see it's already used in this project. :)

Do you like it force-pushed in a single commit or as an additional commit?

Thanks
- Stephan

Reply:
Interestingly, adding the nullability annotation yields a compile error in the default solution.
`AttestationVerificationSuccess.cs(17, 32): [CS8632] The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.`

Reply:
This should work:

```csharp
#nullable enable
public X509Certificate2? AttestationCertificate { get; set; }
#nullable disable
```

Reply:
Of course that would work, however it feels like a big hack...
Will add for now, but maybe the nullability should be enabled on the whole project?

Reply:
@boomer41 yes, nullable will eventually be turned on but we don't have the time to refactor the entire project to support it. So we do it piecewise.